login_test.py

```diff
@@ -129,57 +19,33 @@ def test_cantLoginWithBadCredentials():
     global dummy_user_email
     for driver_constructor in driver_constructor_generator():
         with driver_constructor() as driver:
-            try_login(driver, "thatsnotmypassword")
+            try_login(driver, dummy_user_email, "thatsnotmypassword")
             assert "findes ikke" in driver.page_source
 
 def test_canLoginWithGoodCredentials(dummy_user):

```

Comment:
Can we annotate these tests with the test id that they correspond to (from the tests spreadsheet)?

Reply:
"annotate" in the non-technical sense, just add a comment

Reply:
These aren't yet in the form of the spreadsheet (because we aren't creating users via the signup flow), but I can annotate them with their moral equivalent.